util/components.js

```diff
@@ -16,4 +16,24 @@ export async function setOEBInputValueByCSS(driver, cssSelector, value) {
     const oebInputChild = await oebInputParent.findElement(By.css("input"));
     await oebInputChild.clear()
     await oebInputChild.sendKeys(value);
-}
\ No newline at end of file
+}
+
+export async function clickUntilInteractable(elementCreator) {
+    for (let i = 0; i < 100; i++) {
```

Comment:
An optional parameter for the max number of tries or a constant (=less magic) would be nice here